### ISSUE TITLE
Feature Request: Selective Profiling

### ISSUE THREAD

**Issue by diveyez** [ALiVEOS/ALiVE.OS#595]
In the ALiVE admin actions there is an option to profile non-profiled units which is great, however using it... is a massive pain because that requires us to literally make a mission file with nothing on it. Period.
This function causes problems. For example, I place lots of items down for our missions. If I profile non-profiled units it literally removes them from the battlefield which is unacceptable.

What we need is a way to select which units, and groups we wish to profile, or even perhaps a zeus module that does it because, we only want to profile enemy units, vehicles, civi units, vehicles, etc. I know the sync to ALiVE AI System functions however, what I want to do is profile units that Zeus has put down so large scale missions with Zeus can take advantage of the AI system and its cache while designing the missions by hand. This is a MUST and is very much needed.

We do not want it to profile our own vehicles and bases. For obvious reasons.

**Comment by highhead** [comment]
I believe there is a variable you can set in the editor init-field of the unit! someone beat me to it. An AFAIK re-indexing units that have placed with zeus has also already been handled by ALiVE! Someone beat me to it!

**Comment by diveyez** [comment]
Negative, this is not true. Need a way to selectively profile units without profiling everything. Reasons for this are obvious.

**Comment by marceldev89** [comment]
Yes it is, you should be able to `this setVariable ["ALIVE_profileIgnore", true];` in the unit's/vehicle's init field. Yes it's tedious but it's definitely a thing so that it's ignored by the admin profile action thingy.

**Comment by diveyez** [comment]
That would take 96 hours to do on these missions we run

**Comment by diveyez** [comment]
@marceldev89  
*What sections of ALiVE.os need to be edited to make this a feature in 3den/Zeus properties/attributes for units?*
```
this setVariable ["ALIVE_profileIgnore", true];
```

**Comment by DavisBrown723** [comment]
A function already exists in the code base to accomplish this functionality.

**ALIVE_fnc_createProfilesFromUnitsRuntime**

Parameters
- debug: true/false | defaults to false
- groups: array | defaults to result of sqf command "allGroups"
- vehicles: array | defaults to result of sqf command "vehicles"

Example:
_groupsToProfile = [...]
_vehiclesToProfile = [...]
[false, _groupsToProfile, _vehiclesToProfile ] call ALIVE_fnc_createProfilesFromUnitsRuntime;

A Zeus module that wraps this functionality will be available once https://github.com/ALiVEOS/ALiVE.OS/issues/410 is completed
